src/Parser/Shaman/Elemental/Modules/Talents/LightningRod.js

```diff
@@ -21,21 +20,33 @@ class LightningRod extends Analyzer {
     combatants: Combatants,
   };
 
+  lightningRodDamage=0;
+
   on_initialized() {
     this.active = this.combatants.selected.hasTalent(SPELLS.LIGHTNING_ROD_TALENT.id);
   }
 
+  on_byPlayer_damage(event) {
+    if (event.ability.guid===SPELLS.LIGHTNING_ROD_DAMAGE.id){
+      this.lightningRodDamage+=event.amount;
```

Comment:
Remember to count absorbed damage, should be `this.lightningRodDamage += event.amount + (event.absorbed || 0);`